test/unit/native_deps/native_deps_test.bzl

```diff
@@ -19,17 +19,6 @@ def _native_dep_lib_name(ctx):
     else:
         return "libnative_dep.a"
 
-def _lib_has_no_native_libs_test_impl(ctx):
```

Comment:
What's the deal with this removal? Feels like the error here was probably not adding a test for it, rather than adding a superfluous impl?

Reply:
we no loner use `lib` as a crate type. It's always explicitly `rlib`. To use this test would require adding a new code path to re-implement `lib` which I don't think would be that valuable.

Reply:
https://github.com/bazelbuild/rules_rust/blob/449935861615095c7cc3c1ef477babe7ab48baf4/rust/private/rust.bzl#L170-L183